MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/Constants.java

```diff
@@ -37,6 +40,14 @@
             0.0f, 0.0f, 1.0f
     };
 
+    /**
+     * Coordinates for corners of frame in OpenGL position coordinate system.
+     */
+    public static final Float[] BOTTOM_LEFT  = {-1.0f, -1.0f, 1.0f};
```

Comment:
Is there any reason why you are using Float[] instead of float[] everywhere? Boxed types are way more expensive than their unboxed counterparts.

Reply:
The stabilization data in the video track is passed as a List<Float> to represent each homography, so I just kept using Float for consistency. Should I change everything back to float?